public_html/wp-content/plugins/wordcamp-payments/javascript/wordcamp-budgets.js

```diff
@@ -149,32 +150,6 @@ jQuery( document ).ready( function( $ ) {
 			}
 		},
 
-		/**
-		 * Set the default payment method based on the currency
-		 *
-		 * Don't override any existing payment method choices.
-		 *
-		 * @param {object} event
-		 */
-		setDefaultPaymentMethod : function ( event ) {
```

Comment:
What was the reason for removing this?

Reply:
Sorry I should've added more details for this change. A few reasons to remove this.

1. It kind of didn't work as expected, from my understanding, it the default payment should be checked only when the default a new currency is chosen or there's already a selected payment method. But from I observed, it always default to `Wire` and doesn't change when switching to a different currency.
2. Currently, once the `Country for payment receipt` is selected, the system lists all payment methods based on whether the chosen country is the US. After this, users pick a payment method, and only then does the relevant form for that method become visible. 

Even though the `setDefaultPaymentMethod` might work initially, it ends up auto-selecting either `Wire` or `Direct Deposit` for the user based on the currency they choose, but the desired outcome is to let users choose between `Deposit` or `Check` only when the payment receipt country is set to the US, rather than basing it on the currency.

I've added the above details to the commit message.